Jekyll.gitignore

```diff
@@ -1,3 +1,44 @@
+# Ignore docs files
+.ruby-version
+
+# Numerous always-ignore extensions
+*.diff
+*.err
+*.orig
+*.log
+*.rej
+*.swo
+*.swp
+*.zip
+*.vi
+*~
+
+# OS or Editor folders
```

Comment:
The OS-specific files should be covered by the `Global/OSX.gitignore` and `Global/Windows.gitignore` or other editor-specific template.